### DISCUSSION TITLE
On-Demand Cloud website unavailable mutliple times every day

### DISCUSSION THREAD

**Discussion by deraleks1** [directus/directus#6513] [Archive]
Hey all,
I currently have an on-demand cloud instance subscripted to you. Everything is working out for the most part.
The weird thing is that the website is not available for 15-30 minutes multiple times a day. Checking https://status.directus.cloud/ however delivers that the servers are up. Can I access the application logs and iff yes where can I request it?

Kind regards
:)

**Comment by rijkvanzanten** [comment]
Heya!


This is the repo for the open source Directus project. Support for Directus Cloud can be had through the chat widget in the bottom right of the Cloud Dashboard 👍🏻

**Reply by deraleks1** [reply]
ah oke. thanks ^^
